.prettierignore

```diff
@@ -1 +1,2 @@
 CHANGELOG.md
+/pages/docs/quickstart.mdx
```

Comment:
This would ignore all changes in the file and cause further issues in the future